### PR TITLE
Feature restaurant

### PR DESCRIPTION
**Pull request by atoledo-ec**
## Feature Restaurant

This project includes three pages using JavaScript and also is configured with Webpack.
The page is about a restaurant where you can see a home page, a menu of the food available and the page to contact the restaurant.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
So far so good @alejandrotoledoweb! Nice job working with the ES6 modules! :+1: However, there are some changes for you to make before completing this project. These include:
- [x] Use arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+function setBtnActive(id) {
```
Comment (Forison) [inline src/contact.js]: * [x] Should be converted to arrow function

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  const activeBtn = document.querySelector('.tab.active');
+  if (activeBtn) activeBtn.classList.remove('active');
+
+  const homeBtn = document.getElementById(id);
+  homeBtn.classList.add('active');
+}
+function createSection(name, data) {
```
Comment (Forison) [inline src/contact.js]: * [x] Should be converted to arrow function

File: src/home.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  const para = document.createElement('p');
+  para.classList.add('section-description');
+  para.textContent = 'The best from our cuisine for your taste and enjoy!, Please order want you want as fast as possible!.';
+
+  about.appendChild(para);
+  return about;
+}
+
+function setBtnActive(id) {
```
Comment (Forison) [inline src/home.js]: * [x] Should be converted to arrow function

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  const homeBtn = document.getElementById('home');
+  const menuBtn = document.getElementById('menu');
+  const contactBtn = document.getElementById('contact');
+
+  homeBtn.addEventListener('click', loadHome);
+  menuBtn.addEventListener('click', loadMenu);
+  contactBtn.addEventListener('click', loadContact);
+};
+
+function init() {
```
Comment (Forison) [inline src/index.js]: * [x] Should be converted to arrow function

File: src/initial-page-load.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+function createBtn(id, text) {
```
Comment (Forison) [inline src/initial-page-load.js]: * [x] Should be converted to arrow function

File: src/initial-page-load.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  const btn = document.createElement('button');
+  btn.setAttribute('id', id);
+  const span = document.createElement('span');
+  span.textContent = text;
+  btn.appendChild(span);
+  btn.classList.add('tab');
+  return btn;
+}
+function createNav(id) {
```
Comment (Forison) [inline src/initial-page-load.js]: * [x] Should be converted to arrow function

File: src/initial-page-load.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  menuBtn.className = 'nav-link active nav-item';
+  const contactBtn = createBtn('contact', 'CONTACT');
+  contactBtn.className = 'nav-link active nav-item';
+
+  nav.appendChild(homeBtn);
+  nav.appendChild(menuBtn);
+  nav.appendChild(contactBtn);
+  return nav;
+}
+function createMain(id) {
```
Comment (Forison) [inline src/initial-page-load.js]: * [x] Should be converted to arrow function

File: src/menu.js [hunk 8]
```diff
@@ -0,0 +1,83 @@
+function setBtnActive(id) {
```
Comment (Forison) [inline src/menu.js]: * [x] Should be converted to arrow function

File: src/menu.js [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+  const activeBtn = document.querySelector('.tab.active');
+  if (activeBtn) activeBtn.classList.remove('active');
+
+  const homeBtn = document.getElementById(id);
+  homeBtn.classList.add('active');
+}
+
+function createFood(src, text, price, alt) {
```
Comment (Forison) [inline src/menu.js]: * [x] Should be converted to arrow function

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change required❌
#### Hi Alejandro
#### Good job on addressing partially the issues raised in the previous review. We need to apply that recommendation in all parts of the project🤗.
* [ ] All files in `scr` folder still contain function declared without the arrow function syntax.
### That is it kindly make the changes and submit a re-review
#### Happy coding
Addo Forison

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hi team :hand: 
Great job implementing the requested changes.
Good luck with the next project.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**